Action/vrijbrp.ZgwToVrijbrpAction.action.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "ZgwToVrijbrpAction",
```

Comment:
Keep in mind that this is not the only VrijBRP bundle, maybe specify the name of the action a bit more